### PR TITLE
Add stale config file

### PR DESCRIPTION
**Pull request by FillZpp**
### Ⅰ. Describe what this PR does

Add stale config file.

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 30
+
+# Issues with these labels will never be considered stale
+exemptLabels:
```
Comment (jasonliu747) [inline .github/stale.yml:8]: should we add `hold` & `WIP` in exemptLabels?

**Review by jasonliu747 — COMMENTED** [review]
Nice! I added a stale robot few weeks ago but forgot to upload its config file.

File: .github/stale.yml (hunk 1 above)
Comment (FillZpp) [reply .github/stale.yml:8]: Not sure... Looks like PRs in K8s community will be marked stale and closed even have `hold` or `WIP` labels.

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Pull request merged**